package.json

```
@@ -4,11 +4,12 @@
   "description": "This is going to be the service to retrieve data from all the different sources we are going to have.",
   "main": "server.js",
   "scripts": {
+    "dev": "nodemon node server.js",
```

you can add the prestart script here
```suggestion
    "dev": "npm run prestart && nodemon node server.js",
```

---

Fixed